src/views/forms/NewMarketForm/v7/NewMarketPreviewStep.tsx

```diff
@@ -58,6 +68,25 @@ export const NewMarketPreviewStep = ({
   const { createPermissionlessMarket } = useSubaccount();
   const { usdcImage } = useTokenConfigs();
   const { freeCollateral } = useAppSelector(selectSubaccountStateForVaults);
+  const marketOraclePrice = useAppSelector((s) => getMarketOraclePrice(s, ticker));
+  const [txHash, setTxHash] = useState<string>();
```

Comment:
huuuh surely the types lie.

Reply:
<img width="313" alt="Screen Shot 2024-10-28 at 12 12 45 PM" src="https://github.com/user-attachments/assets/c42c4097-e0e0-4b08-b78a-ed4e2d57ef13">

If the initialValue is undefined it knows!

Reply:
maaaaaagic. Nice!